test/backend/models.query.incident-query.test.js

```diff
@@ -15,10 +15,12 @@ describe('Incident query attributes', function() {
     Incident.remove(function(err) {
       if (err) return done(err);
       async.each([
-        { title: 'The quick brown fox', veracity: null, closed: true },
-        { title: 'The slow white fox', veracity: true, closed: false },
-        { title: 'The quick brown chicken', veracity: null, closed: false },
-        { title: 'The brown quick fox', veracity: false, closed: true }
+        { title: 'The quick brown fox', veracity: null, closed: true, tags: ['hello', 'world'] },
+        { title: 'The slow white fox', veracity: true, closed: false, tags: ['hello'] },
+        { title: 'The quick brown chicken', veracity: null, closed: false, tags: ['world'] },
+        { title: 'The brown quick fox', veracity: false, closed: true, tags: ['helloworld', 'wellohorld', 'foobar'] },
+        { title: 'The fox that was slow and brown', veracity: false, closed: true, tags: ['foobar', 'hello'] },
+        { title: 'The slow that was fox and brown', veracity: false, closed: true, tags: ['wellohorld', 'hello'] }
```

Comment:
I just followed the format of what was already there. I'll make this change, as it looks better.